json-rpc/tests/integration_test.rs

```diff
@@ -1369,6 +1370,22 @@ fn create_test_cases() -> Vec<Test> {
                 assert_eq!(events.len(),3);
             },
         },
+        Test {
+            name: "get_event_by_version_with_proof",
+            run: |env: &mut testing::Env| {
+                let event_key = diem_types::block_metadata::new_block_event_key();
+                let response = env.send(
+                    "get_event_by_version_with_proof",
+                    json!([event_key.to_string()]),
+                );
+
+                // Just check that the responses deserialize correctly, we'll let
+                // the verifying client smoke tests handle the proof checking.
+                let value = response.result.unwrap();
+                let view = serde_json::from_value::<EventByVersionWithProofView>(value).unwrap();
+                let _proof = EventByVersionWithProof::try_from(&view).unwrap();
+            },
+        },
```

Comment:
You're going to have to rebase and implement this using the forge interface (since I recently had a pr land to convert all the jsonrpc integration tests)

Reply:
done!